templates/base.html

```diff
@@ -47,102 +48,114 @@
+                                <li class="nav"><a href="{% url 'settings' %}"> <i class="fa fa-gears" aria-hidden="true"></i> SETTINGS</a></li>
+                                <li class="nav"><a href="{% url 'logout' %}"><i class="fa fa-sign-out" aria-hidden="true"></i>LOGOUT</a></li>
+                            </ul>
+                        </li>
+                    </ul>
                 </div>
             </div>
         </div>
 
+        {% else %} 
+        <div class="top-bar nav">
+            <div class="top-bar-left">
+                <ul class="menu nav">
+                    <li class="home-logo"><a class="home-logo" href="{% url 'home' %}"><img class="home-logo" src="{% static "img/perdiem-logo.svg" %}"></a></li>
+                </ul>
+            </div>
+            <div class="top-bar-right login-item">
+                <ul class="menu">
+                    <li class="nav"><a class="" href="{% url 'register' %}?next={{ request.get_full_path }}">SIGN UP</a></li>
+                    <li class="nav"><a class="" data-open="login-modal">LOGIN</a></li>
+                </ul>
+            </div>
+        </div>
+        {% endif %}
+
+        <!-- Content -->
+        <div class="container page">
+            {% block content %}
+            {% endblock %}
+        </div>
+
+        <!-- Footer -->
+        <div class="footer">
+            <ul class="footer-items">
+                <li><a href="{% url 'privacy' %}">Privacy</a></li>
+                <li><a href="{% url 'terms' %}">Terms of Use</a></li>
+                <li><a href="{% url 'resources' %}">Resources</a></li>
+                <li><a href="{% url 'faq' %}">FAQ</a></li>
+                <li><a href="{% url 'contact' %}">Contact</a></li>
+            </ul>
+            <ul class="social-icons">
+                <li><a href="https://www.facebook.com/828188463946021" target="_blank"><i class="fa fa-facebook fa-2x" aria-hidden="true"></i></a></li>
+                <li><a href="http://instagram.com/perdiemmusic" target="_blank"><i class="fa fa-instagram fa-2x" aria-hidden="true"></i></a></li>
+                <li><a href="http://twitter.com/perdiemmusic" target="_blank"><i class="fa fa-twitter fa-2x" aria-hidden="true"></i></a></li>
+                <li><a href="https://www.youtube.com/channel/UC-6u9L-W6Dn7kaYLOz4YuTg" target="_blank"><i class="fa fa-youtube fa-2x" aria-hidden="true"></i></a></li>
+            </ul>
+        </div>
+
         <!-- Success Modal -->
         <div class="reveal" id="success-modal" data-reveal>
             <h1 class="white-color center">DONE</h1>
             <p class="success-message white-color center">The action completed successfully.</p>
             <div class="success-extra white-color"></div>
             <div class="center">
-                <button class="button center" data-close>Okay, got it!</button>
+                <a class="button center" data-close href="{% url 'profile' %}">Okay, got it!</a>
```

Comment:
Would it make more sense to create a modal specifically for after investing? I really want people to go to their profile after investing so they can see their artists, music, updates, etc.. 

Right now after people click okay they just scroll around the page and then eventually leave. Most people don't even know they have a profile.